src/current/v22.1/import-performance-best-practices.md

```diff
@@ -5,28 +5,89 @@ toc: true
 docs_area: migrate
 ---
 
-This page provides best practices for optimizing [import](import.html) performance in CockroachDB.
+This page provides best practices for optimizing [import](import-into.html) performance in CockroachDB.
 
-Import speed primarily depends on the amount of data that you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+`IMPORT INTO` is the fastest method to ingest data into CockroachDB but it requires taking the target table offline for the duration of the import. `IMPORT INTO` is a good choice for initial data migrations and data migrations that can tolerate table downtime. If you cannot tolerate table unavailability, we recommend using [`COPY FROM`](copy-from.html) instead.
 
-- [Splitting data](#split-your-data-into-multiple-files)
-- [Import format](#choose-a-performant-import-format)
+Import performance primarily depends on the amount of data that you want to import. However, there are three actions you can take before importing that have a significant impact on the amount of time it will take to run an import:
```

Comment:
```suggestion
Import performance primarily depends on the amount of data that you want to import. However, there are three actions you should take before importing that will have a significant impact on the amount of time it takes to run an import:
```
Minor wording suggestion to make this a stronger suggestion. Feel free to ignore